.php_cs

```diff
@@ -37,12 +38,14 @@ $config->fixers(
         'function_call_space',
         'function_declaration',
         'indentation',
+        'join_function',
```

Comment:
Maybe we should bump the `composer.json` version of the fixer, to avoid regressions?